src/test/scala/uk/gov/nationalarchives/notifications/LambdaSpecUtils.scala

```diff
@@ -205,12 +218,18 @@ class LambdaSpecUtils extends AnyFlatSpec with Matchers with BeforeAndAfterAll w
   }
 
   def exportStatusSlackBody(exportStatusEvent: ExportStatusEvent): String = {
+    val successDetails = exportStatusEvent.successDetails
+    val failureCause = exportStatusEvent.failureCause
+    val exportOutputMessage = if(successDetails.isDefined) {
+      s""":\\nUser ID: ${successDetails.get.userId}\\nConsignment Reference: ${successDetails.get.consignmentReference}\\nTransferring Body Code: ${successDetails.get.transferringBodyCode}"""
+    } else if(failureCause.isDefined) s""":\\nCause: ${failureCause.get}""" else """"""
+
     s"""{
        |  "blocks" : [ {
        |    "type" : "section",
        |    "text" : {
        |      "type" : "mrkdwn",
-       |      "text" : "The export for the consignment ${exportStatusEvent.consignmentId} has ${if (exportStatusEvent.success) "completed" else "failed"} for environment ${exportStatusEvent.environment}"
+       |      "text" : "The export for the consignment ${exportStatusEvent.consignmentId} has ${if (exportStatusEvent.success) "completed" else "failed"} for environment ${exportStatusEvent.environment}${exportOutputMessage}"
```

Comment:
This isn't new in your PR, but these tests are duplicating quite a lot of the the logic that they're supposed to be testing, which means they're less likely to actually catch a bug.

I'm going to make a note of this, because I'd quite like to revisit them and see if we can reduce the logic.

Reply:
Agree. It was pretty fiddly getting updating these tests.